MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+
+Once the version branch is created, the patch is either cherry picked or backported into a PR against the version branch. The version branch is maintained via the same process as the master branch, including PR approval process etc.
+
+Releases for the previous major version are made directly from its version branch. Ensure that the `CHANGELOG` is updated in both the master and the version branch to reflect the new release.
+
+### Releasing
+The SPIRE release machinery is tag-driven. When the maintainers are ready to release, a tag is pushed referencing the release commit. While the CI/CD pipeline takes care of the rest, it is important to keep an eye on its progress. If an error is encountered during this process, the release is aborted.
+
+The first two releases that a new maintainer performs must be performed under the supervision of maintainer that has already satisfied this requirement.
+
+SPIRE releases are authorized by its maintainers. When doing so, they should carefully consider the proposed release commit. Is there confidence that the changes included do not represent a compatibility concern? Have the affected codepaths been sufficiently exercised, be it by automated test suite or manual testing? Is the maintainer free of general hesitation in releasing this commit, particularly with regards to safety and security? If the answer to any of these questions is "no", then do not release.
+
+A simple majority vote is required to authorize a SPIRE release at a specific commit hash. If any maintainer feels that the result of this vote critically endangers the project or its users, they have the right to raise the matter to the SPIFFE TSC. If this occurs, the release in question MUST be frozen until the SPIFFE TSC has made a decision. To say that this would be unprecedented is an understatement. Do not take this route lightly.
+
+#### Checklist
+This section summarizes the steps necessary to execute a SPIRE release. Unless explicitly stated, the below steps must be executed in order.
+
+The following steps must be completed at least one week prior to release:
+* Ensure all changes intended to be included in the release are fully merged
+* Identify a specific commit as the release candidate
+* Raise an issue "Release SPIRE X.Y.Z", and include both the release candidate commit hash and the proposed changelog updates
+
+**If this is a major release**, the following steps must be completed before releasing:
+* Review and exercise all examples in spiffe.io and spire-examples repo against the release candidate hash
```

Comment:
may we add spire-tutorial to this list?

Reply:
This is covered under the spiffe.io umbrella as updating it would include updating spire-tutorials.